### ISSUE TITLE
Split OVAs into different releases and other release process changes

### ISSUE THREAD

**Issue by justinzw** [Seagate/cortx#441]
We should change the way we mark releases and not bundle a bunch of OVAs into one "release"
https://github.com/Seagate/cortx/releases/tag/VA

For example let's look at https://github.com/tensorflow/tensorflow/releases

A few things they do that I feel we should:

1) They mark the latest release with the tag "Latest Release"
![image](https://user-images.githubusercontent.com/950172/97304440-f1cda500-1818-11eb-80df-db76728fe960.png)

2) Each "release" has one folder/zip file. Not multiple.

3) They say what is in each release with a list of bugs/features

+ @johnbent @shailesh-vaidya @gregnsk @mukul-seagate11 @amolkongre 
What do you all think?

**Comment by shailesh-vaidya** [comment]
@just-now  Yes we should generate GitHub release for each new OVA.  We should also consider below points,

- Versioning -  We used [VA](https://github.com/Seagate/cortx/releases/tag/VA) and [JBOD](https://github.com/Seagate/cortx/releases/tag/JBOD) tag names for current releases. We should use [semantic versioning](https://semver.org/) for generating release tag. 
- Release Notes - We should add release notes with Improvements, Bug fixes etc.  for every release. 
- Need to add above tasks in OVA generation automation process.

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya, please check & update

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya, please check so that this request can be forwarded?

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya, please check & close once you are back from holidays

**Comment by shailesh-vaidya** [comment]
- OVA Build frequency -  Every Sprint End ( 2 Weeks)

- OVA Versioning -   As mentioned in https://github.com/Seagate/cortx/issues/441#issuecomment-717253653  We should use semantic versioning for OVA . Currently we have released cortx-va-1.0.*.zip file at https://github.com/Seagate/cortx/releases/tag/VA .  We should create new GitHub releases names cortx-va-1.0.`Incremental number`  and upload cortx-va-1.0.`incremental number`.zip into it. 

- Release Notes -  We can add Jira query or GitHub log query output to list down changes landed into main branch in sprint. For internal releases we used to send release notes with Jira issues fixed within sprint along with Provisioning guide. @Venkataraman has added Release Notes template at https://github.com/Seagate/cortx/blob/main/doc/RNTemplate.rst . We can improve and add it as description to GitHub release.
